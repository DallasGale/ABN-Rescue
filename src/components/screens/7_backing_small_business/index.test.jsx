/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import BackingSmallBusiness from '.'



describe('<BackingSmallBusiness />', () => {
  it('renders without crashing', () => {
    const component = render(<BackingSmallBusiness duration={5000} />)
    expect(component).toMatchSnapshot()
  })
})
