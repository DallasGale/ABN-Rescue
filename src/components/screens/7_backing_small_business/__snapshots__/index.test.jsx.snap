// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackingSmallBusiness /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styled__StyledWrapper-sc-1scmett-1 hEjPYQ"
        style=""
      >
        <div
          class="scrollmagic-pin-spacer"
          data-scrollmagic-pin-spacer=""
          style="display: block; position: relative; box-sizing: content-box; width: 100%; min-height: 0px; height: auto; padding-top: 0px; padding-bottom: 5000px;"
        >
          <div
            class="sticky"
            style="position: relative; margin: auto; box-sizing: border-box; width: 100%;"
          >
            <div
              class="styled__StyledContent-sc-1scmett-0 jIynGk"
            >
              <h2
                class="typography__StyledMessageH2-sc-1n1kkhl-0 styled__StyledH2-sc-1scmett-2 haBTk"
              >
                For over 165 years,
              </h2>
              <h2
                class="typography__StyledMessageH2-sc-1n1kkhl-0 styled__StyledH2-sc-1scmett-2 haBTk"
              >
                CGU has been insuring ambitious aussies.
              </h2>
              <div
                class="panel top"
                style="top: -90%;"
              >
                <h2
                  class="typography__StyledMessageH2-sc-1n1kkhl-0 message message__top disappear styled__StyledRescueH2-sc-1scmett-3 fhRriM"
                  style="transform: translate(0%,-130%) matrix(1,0,0,1,0,0);"
                >
                  Our ambition?
                </h2>
              </div>
              <div
                class="panel bottom"
                style="bottom: 0px;"
              >
                <h2
                  class="typography__StyledMessageH2-sc-1n1kkhl-0 message message__bottom disappear styled__StyledRescueH2-sc-1scmett-3 fhRriM"
                  style="top: 100%;"
                >
                  To rescue australia's ambition.
                </h2>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styled__StyledWrapper-sc-1scmett-1 hEjPYQ"
      style=""
    >
      <div
        class="scrollmagic-pin-spacer"
        data-scrollmagic-pin-spacer=""
        style="display: block; position: relative; box-sizing: content-box; width: 100%; min-height: 0px; height: auto; padding-top: 0px; padding-bottom: 5000px;"
      >
        <div
          class="sticky"
          style="position: relative; margin: auto; box-sizing: border-box; width: 100%;"
        >
          <div
            class="styled__StyledContent-sc-1scmett-0 jIynGk"
          >
            <h2
              class="typography__StyledMessageH2-sc-1n1kkhl-0 styled__StyledH2-sc-1scmett-2 haBTk"
            >
              For over 165 years,
            </h2>
            <h2
              class="typography__StyledMessageH2-sc-1n1kkhl-0 styled__StyledH2-sc-1scmett-2 haBTk"
            >
              CGU has been insuring ambitious aussies.
            </h2>
            <div
              class="panel top"
              style="top: -90%;"
            >
              <h2
                class="typography__StyledMessageH2-sc-1n1kkhl-0 message message__top disappear styled__StyledRescueH2-sc-1scmett-3 fhRriM"
                style="transform: translate(0%,-130%) matrix(1,0,0,1,0,0);"
              >
                Our ambition?
              </h2>
            </div>
            <div
              class="panel bottom"
              style="bottom: 0px;"
            >
              <h2
                class="typography__StyledMessageH2-sc-1n1kkhl-0 message message__bottom disappear styled__StyledRescueH2-sc-1scmett-3 fhRriM"
                style="top: 100%;"
              >
                To rescue australia's ambition.
              </h2>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
